### ISSUE TITLE
Support DELETE method

### ISSUE THREAD

**Issue by dandv** [Preposterous/twitter-lite#31]
We only support `.get()` and `.post()` now, but there's also [DELETE](https://developer.twitter.com/en/docs/direct-messages/welcome-messages/api-reference/delete-welcome-message-rule.html).

**Comment by CannibalKush** [comment]
Well shit, went and built 80% of my API with this library but can't use Delete.

Why even use this library lol

**Comment by dandv** [comment]
This is how open source works. There has been insufficient demand + developer time to implement delete.

You're free to implement it (the source code is well-commented, and you can have a look at the [PUT PR](https://github.com/draftbit/twitter-lite/pull/43)), or use another library.

**Comment by CannibalKush** [comment]
Yeah sorry i wrote that issue in frustration. i just might implement it actually.

**Comment by dylanirlbeck** [comment]
I'm going to try and tackle this issue. @dandv If there's anything I should know about DELETE and the Twitter API, please let me know!

I'll try to have a PR out soon.

**Comment by camergardi** [comment]
@dylanirlbeck Any update on this?

**Comment by LoSunny** [comment]
It's been some days since this issue, anyone got any updates?
